go.mod

```diff
@@ -8,9 +8,9 @@ go 1.14
 
 // replace github.com/openziti/fabric => ../fabric
 
-// replace github.com/openziti/sdk-golang => ../sdk-golang
+replace github.com/openziti/sdk-golang => ../sdk-golang
```

Comment:
this is an oversight fyi

Reply:
To repeat my top-level into comment:  I will update go.mod once upstream deps have landed and been tagged.

I thought I'd get the actual code eye-balled in the meanwhile.